sample_data/organizations.json

```diff
@@ -0,0 +1,87 @@
+[
+  {
+    "_id": "6437904485008f171cf29924",
+    "status": "ACTIVE",
+    "members": ["64378abd85008f171cf2990d"],
+    "admins": ["64378abd85008f171cf2990d"],
+    "groupChats": [],
+    "posts": [],
+    "pinnedPosts": [],
+    "membershipRequests": [],
+    "blockedUsers": [],
+    "name": "Test Organization 1",
```

Comment:
1. Give the organizations real sounding names, descriptions and locations.
2. Don't use religious names as that may connote bias in a sensitive area. Don't use Palisadoes either.
3. You can end the names with `Charities`, `Foundation` etc. to make them universal.

Reply:
Do this for all organizations.